htmlmaker_postprocessing.js

```diff
@@ -95,6 +106,83 @@ fs.readFile(file, function editContent (err, contents) {
     $(this).after(match[4]);
   });
 
+  // remove Section-Blank-Page sections
+  var sectionBlankPage = $("section.blankpage");
+  sectionBlankPage.remove();
+
+  //// Strip pageBreaks preceding Section starts: Part 1 (of 2):
+  // catch & remove any page break directly preceding Section Starts
+  //  (nothing should be outside of a seciton block, but just in case)
+  var leadingPageBreak = $("section").prev(".PageBreakpb");
```

Comment:
Actually, also make sure to add div handling, so: $(".PageBreakpb + section, .PageBreakpb + div").prev().remove();